index.html

```diff
@@ -76,5 +76,7 @@
         <script src="js/vendor/bootstrap.min.js"></script>
         <script src="js/aggregator.js"></script>
         <script src="js/main.js"></script>
+        <script src="http://cdnjs.cloudflare.com/ajax/libs/moment.js/2.9.0/moment.min.js"></script>
+        <script src="http://cdnjs.cloudflare.com/ajax/libs/angular-moment/0.9.0/angular-moment.min.js"></script>
```

Comment:
@stephendonner, I pinned these two, especially since Angular-Moment 2 has been released, but it's not stable and recommended using version one. 

I haven't done enough research into both of these two plugins to be able to tell whether or not they'll be backwards compatible.